documentation/src/main/asciidoc/userguide/chapters/transactions/Transactions.adoc

```diff
@@ -77,18 +77,9 @@ If that resolution does not work, or if you wish to provide a custom implementat
 Hibernate provides many implementations of the `JtaPlatform` contract, all with short names:
 
 `Atomikos`:: `JtaPlatform` for Atomikos.
-`Borland`:: `JtaPlatform` for the Borland Enterprise Server.
-`Bitronix`:: `JtaPlatform` for Bitronix.
 `JBossAS`:: `JtaPlatform` for Arjuna/JBossTransactions/Narayana when used within the JBoss/WildFly Application Server.
 `JBossTS`:: `JtaPlatform` for Arjuna/JBossTransactions/Narayana when used standalone.
-`JOnAS`:: `JtaPlatform` for JOTM when used within JOnAS.
-`JOTM`:: `JtaPlatform` for JOTM when used standalone.
-`JRun4`:: `JtaPlatform` for the JRun 4 Application Server.
-`OC4J`:: `JtaPlatform` for Oracle's OC4J container.
-`Orion`:: `JtaPlatform` for the Orion Application Server.
 `Resin`:: `JtaPlatform` for the Resin Application Server.
-`SapNetWeaver`:: `JtaPlatform` for the SAP NetWeaver Application Server.
-`SunOne`:: `JtaPlatform` for the SunOne Application Server.
```

Comment:
Should probably be retained and maybe also mention the Glassfish/Payara variants?